src/main/fc/runtime_config.h

```diff
@@ -45,9 +46,10 @@ typedef enum {
     ARMING_DISABLED_CMS_MENU    = (1 << 8),
     ARMING_DISABLED_OSD_MENU    = (1 << 9),
     ARMING_DISABLED_BST         = (1 << 10),
+    ARMING_DISABLED_NOPREARM    = (1 << 11)
```

Comment:
I would move this up s bit, and put it maybe after angle. Counting the number of beeps is hard, therefore the less likely conditions should go towards the end.